src/color/css.ts

```diff
@@ -5,15 +5,36 @@ import {
     HSLObject,
     CMYKObject
 } from '@types';
-import {
-    toHEX,
-    hasProp,
-    round
-} from '#helpers';
+import { toHEX, hasProp, round, roundAll, toHEXAll } from '#helpers';
```

Comment:
I prefer more than two imports in different lines, even if Prettier don‘t do it.